### PR TITLE
Add ca-policy-lcg

### PR DESCRIPTION
**Pull request by chrisburr**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/ca-policy-lcg/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+build:
+  noarch: generic
+  number: 0
+  # DIRAC will try to update these automatically so avoid linking between environments
+  no_link:
+    - ${PREFIX}/etc/grid-security/certificates/*
+
+requirements:
+  build:
+    # Dummpy requirement to ensure conda-forge builds this correctly
```
Comment (chrisburr) [inline recipes/ca-policy-lcg/meta.yaml]: I've opened an issue about this (https://github.com/conda-forge/staged-recipes/issues/8377)

**Comment by chrisburr** [comment]
@conda-forge/staged-recipes This is ready for review.

These are normally distributed as many RPMs combined as several metapackages for different applications. I've combined them into one here to make it easier to maintain. It can always be split up in future if required.

File: recipes/ca-policy-lcg/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+    - make
+
+test:
+  commands:
+    - echo "${X509_CERT_DIR}"
+
+about:
+  home: http://www.egi.eu/
+  license: IGTF Federation
+  license_file: LICENSE
```
Comment (scopatz) [inline recipes/ca-policy-lcg/meta.yaml]: You could add `license_family: other`

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**